src/quizImage/Image.java

```diff
@@ -1,13 +1,13 @@
-import javax.imageio.ImageIO;
+package quizImage;

```

Comment:
I'm not 100% sure, but package names should not contain uppercased letters. You can name it just image, or quizimage or quiz_image. 

Or instead of that make packages equation.image and equation.quiz for second package.